resources/user_config.js

```diff
@@ -39,8 +39,20 @@ let UserConfig = {
 
       // The plugin auto-detects the language, so set this first.
       // If options files want to override it, they can for testing.
-      if (e.detail.language)
-        Options.Language = e.detail.language;
+      if (e.detail.parserLanguage) {
+        Options.ParserLanguage = e.detail.parserLanguage;
+        // Backward compatibility, everything "Language" should be changed to "ParserLanguage"
+        Options.Language = e.detail.parserLanguage;
+      }
+      if (e.detail.systemLocale) {
+        Options.SystemLocale = e.detail.systemLocale;
+        Options.ShortLocale = e.detail.SystemLocale.substring(0, 2);
+        if (Options.ShortLocale == 'zh')
+          Options.ShortLocale = 'cn';
+        const supportedLanguage = ['en', 'de', 'fr', 'ja', 'cn', 'ko'];
+        if (supportedLanguage.indexOf(Options.ShortLocale) == -1)
```

Comment:
Funnily enough, #1408 is adding this list.  I think this is fine for now, and we can merge this to use the other later.

Can you also make this `if (Options.ShortLocale.includes(supportedLanguage))` instead of using `indexOf`?

Reply:
I think this is supposed to be `if (supportedLanguage.includes(Options.ShortLocale))`?

Reply:
Oops thanks, sorry.  I really just meant "use includes" here.

Reply:
Is there any reason to use `includes`?
checking `indexOf` is way more faster than `includes`
![Screenshot 2020-05-06 at 13 05 49](https://user-images.githubusercontent.com/4695371/81137621-5f032a80-8f9a-11ea-81b7-8d3ffdcc7637.jpg)
tested from: http://jindo.dev.naver.com/jsMatch/index.html